### PR TITLE
DEV: Add Ghanaian holidays

### PR DESCRIPTION
**Pull request by s3lase**
Add and generate definitions for Ghanaian holidays

### CODE REVIEW

**Review by ZogStriP — APPROVED** [review]

File: vendor/holidays/definitions/gh.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    observed: to_monday_if_weekend(date)
+  - name: Christmas Day
+    regions: [gh]
+    mday: 25
+  - name: Boxing Day
+    regions: [gh]
+    mday: 26
+    observed: to_weekday_if_boxing_weekend(date)
+
+tests:
```
Comment (ZogStriP) [inline vendor/holidays/definitions/gh.yaml:73]: Excellent, you even added tests 👍

File: vendor/holidays/lib/generated_definitions/REGIONS.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
...es_cn, :es_ct, :es_ex, :es_ga, :es_ib, :es_lo, :es_m, :es_mu, :es_na, :es_o, :es_pv, :es_v, :es_vc, :federalreserve, :federalreservebanks, :fedex, :fi, :fr, :fr_a, :fr_m, :gb, :gb_con, :gb_eaw, :gb_eng, :gb_gsy, :gb_iom, :gb_jsy, :gb_nir, :gb_sct, :gb_wls, :ge, :gg, :hk, :hr, :hu, :id, :ie, :im, :in, :is, :it, :it_bl, :it_fi, :it_ge, :it_pd, :it_rm, :it_ro, :it_to, :it_tv, :it_ve, :it_vi, :it_vr, :je, :jp, :kr, :kz, :li, :lt, :lu, :lv, :ma, :mt_en, :mt_mt, :mx, :mx_pue, :my, :nerc, :ng, :nl, :no, :nyse, :nz, :nz_ak, :nz_ca, :nz_ch, :nz_hb, :nz_mb, :nz_ne, :nz_nl, :nz_ot, :nz_sc, :nz_sl, :nz_ta, :nz_we, :nz_wl, :pe, :ph, :pl, :pt, :pt_li, :pt_po, :ro, :rs_cyrl, :rs_la, :ru, :sa, :se, :sg, :si, :sk, :th, :tn, :tr, :ua, :unitednations, :ups, :us, :us_ak, :us_al, :us_ar, :us_az, :us_ca, :us_co, :us_ct, :us_dc, :us_de, :us_fl, :us_ga, :us_gu, :us_hi, :us_ia, :us_id, :us_il, :us_in, :us_ks, :us_ky, :us_la, :us_ma, :us_md, :us_me, :us_mi, :us_mn, :us_mo, :us_ms, :us_mt, :us_nc, :us_nd, :us_ne, :us_nh, :us_nj, :us_nm, :us_nv, :us_ny, :us_oh, :us_ok, :us_or, :us_pa, :us_pr, :us_ri, :us_sc, :us_sd, :us_tn, :us_tx, :us_ut, :us_va, :us_vi, :us_vt, :us_wa, :us_wi, :us_wv, :us_wy, :ve, :vi, :za]
```
Comment (ZogStriP) [inline vendor/holidays/lib/generated_definitions/REGIONS.rb:3]: Not sure why the lists are so different 🤔 

Can you do a quick diff between the old and new to see what has been added/removed?

Comment (s3lase) [reply vendor/holidays/lib/generated_definitions/REGIONS.rb:3]: Sure

Comment (s3lase) [reply vendor/holidays/lib/generated_definitions/REGIONS.rb:3]: These regions are missing `us_az, us_co, us_oh, us_or, us_sd, us_wy`. 

It appears they were only ever explicitly used for one one holiday. It was updated here https://github.com/discourse/discourse-calendar/commit/3e67f009c1fca577521e2c00a52756744f43861e

As to why it's now showing up as a change here, perhaps the definitions weren't generated for that change?

Comment (ZogStriP) [reply vendor/holidays/lib/generated_definitions/REGIONS.rb:3]: Nice catch. Yeah, we probably forgot to generate/update the definitions.

Looks like those regions are still referenced in `assets/javascripts/lib/regions.js`, which needs to be updated using the `rake javascript:update_constants` task.

Can you do that as well?

Comment (s3lase) [reply vendor/holidays/lib/generated_definitions/REGIONS.rb:3]: Sure

Comment (s3lase) [reply vendor/holidays/lib/generated_definitions/REGIONS.rb:3]: Done

**Review by ZogStriP — APPROVED** [review]

**Pull request merged**